### ISSUE TITLE
warning in pprldmany with function=<closed file...

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#1496]
```
cocopp/compall/pprldmany.py:823: UserWarning: The data for algorithm ./exdata/fmin_on_bbob_budget0002xD-001 and function <closed file 'ppdata/rando_fmin/pprldmany_03D_separ.tex', mode 'w' at 0x110bb9a50> are missing
  warnings.warn('The data for algorithm %s and function %s are missing' % (alg, f))
```
is a bug, because `f` is not the right variable to print here.

**Comment by islandblue** [comment]
Earlier in the code, f is assigned what I believe to be a function number/index, but before this line, it is reassigned a file object. Sadly, I wasn't able to figure out what the correct expression should be, due to lack of knowledge on the data structures involved. I'll leave this to someone more familiar with the code in question.

**Comment by nikohansen** [comment]
Using a different variable name for the file should probably do then.

**Comment by brockho** [comment]
@islandblue remarks that the warning is issued much after the function `f` is defined (within an already finished loop) and we might also consider moving the warning.

**Comment by brockho** [comment]
Seems to be closed with #1547 already.
